src/components/Common/TitleAndListWithOptionalLogos.js

```diff
@@ -16,6 +17,17 @@ const ItemBody = styled(Body)`
     padding: 0;
   }
 `
+
+const SquareImg60 = styled(Image)`
+  max-width: 60px;
```

Comment:
We usually like to use remcalc
`import remcalc from 'remcalc'`
and then you can update all your `XXpx` to `${remcalc(XX)}` instead.

Reply:
I believe its ok to use hardcoded values here. things like padding etc. should be remcalc etc.

Reply:
wait sorry @GBouffard is totally right, ignore me 🤦‍♂

Reply:
Going to leave hardcoded values for now.